src/CMakeLists.txt

```diff
@@ -399,10 +399,14 @@ add_subdirectory(bindings)
 set(LIB_INSTALL_DIR lib${LIB_SUFFIX}) # determine correct lib folder
 
 # Install primary library
-INSTALL(  TARGETS openshot
```

Comment:
This also looks suspect to me. Perhaps @ferdnyc would have some thoughts. In theory, cmake should be doing this type of stuff automatically... with respects to handling Mac, Windows, and Linux in a single way... vs having lots of IF WIN, IF MAC type logic all over the place. But I would love to hear what @ferdnyc thinks.

Reply:
Don't know what it should, but I don't see my recently compiled .dll file in the destination folder of the installation path.
The idea behind this was to build (and it does) and copy the .dll (not only .dll.a) and insert it into frozen OpenShot, just to not recompile whole project each time I need to check if my new lovely feature works.

Reply:
Few thoughts from @ferdnyc , similar code https://github.com/OpenShot/libopenshot-audio/pull/52 ^_^

Reply:
Yeah, the DLL not being installed was the missing `RUNTIME DESTINATION` path in the `install()`, which I've submitted PRs to fix in both projects.

If a _single_ destination is specified `install(TARGETS openshot DESTINATION $path)`, everything gets installed there. But if the various products of the target are routed with `ARCHIVE DESTINATION`, `LIBRARY DESTINATION`, etc. then any not specified _aren't_ installed. So we can't leave out `RUNTIME DESTINATION` for DLLs.

But there's absolutely no need to make it conditional by OS.
```
install(TARGETS openshot
		ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
		LIBRARY DESTINATION ${LIB_INSTALL_DIR}
		RUNTIME DESTINATION ${LIB_INSTALL_DIR} )
```
(or, better, `${CMAKE_INSTALL_LIBDIR}` from GnuInstallDirs) can be used on all three OSes.

Reply:
@ferdnyc for what reason are you set 3 **same** paths in a single rule?

Reply:
They control where different products of the target will be installed, in different scenarios, on different OSes. (We just happen to install them all to the same place.) [CMake install():](https://cmake.org/cmake/help/v3.0/command/install.html)
> Static libraries are always treated as `ARCHIVE` targets. Module libraries are always treated as `LIBRARY` targets. For non-DLL platforms shared libraries are treated as `LIBRARY` targets, except that those marked with the `FRAMEWORK` property are treated as `FRAMEWORK` targets on OS X. For DLL platforms the DLL part of a shared library is treated as a `RUNTIME` target and the corresponding import library is treated as an `ARCHIVE` target.

Technically we only care about `LIBRARY DESTINATION` (for Linux `.so` and OS X `.dylib`) and `RUNTIME DESTINATION` (for Windows `.dll`). The `ARCHIVE DESTINATION` will get the Windows `.dll.a` file, though, and would also get any static libraries built on Linux/Mac. (`FRAMEWORK DESTINATION` is also a possibility if we ever build the MacOS library as a Framework, though I know next-to-nothing about that.)

The `RUNTIME DESTINATION` actually _should be_ `${CMAKE_INSTALL_BINDIR}`, so that Windows DLLs would be installed alongside executables in the `bin/` directory. That's how Windows really expects things to be set up, and it's a "more correct" layout. OpenShot on Windows is ultimately packaged that way by cx_freeze, in fact: all of the `.dll`s are placed in the `C:\Program Files\OpenShot Video Editor\` directory alongside the `.exe`. 

But for now, the DLL is still being routed into the library dir, because the previous single-path `install()` command placed it there, so that's the assumption all of the packaging scripts and etc. were written with.

Reply:
@ferdnyc I see only single path for all (despite it written 3 times). And I think that the 3 paths - is just none existing future.

About OS specific code. I did'n knew why there were different install scripts (libopenshot and libopenshot-audio) for one application, so just escaped unknown for me part with the OS specific code, that's all. Now I'm sure that this is just rudiment of the long development.

Reply:
As for the #262 , when it will be implemented (in case it happens first) and there will be no problems with building in the  environment that was mentioned earlier - I will remove this part from the PR too.

Reply:
#262 is merged